openslides_backend/action/actions/meeting/clone.py

```diff
@@ -89,9 +89,10 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
 
         # needs an empty map for superclass code
         self.merge_user_map = {}
-        for id_ in self.get_meeting_from_json(meeting_json).get("user_ids", []):
+        imported_users_ids = [int(key) for key in meeting_json.get("user", {})]
```

Comment:
Couldn't you set self.merge_user_map directly with a dict comprehension like
`self.merge_user_map = { id_ : id_ for key in ...}`
and for the len you could use `len(self.merge_user_map)`

Reply:
Used, int of key, because the key is a string.